pkg/monitortests/node/nodestateanalyzer/nodeTest/drain-02/startingEvents.json

```diff
@@ -2,31 +2,101 @@
   "items": [
     {
       "level": "Info",
-      "locator": "ns/openshift-machine-config-operator node/ci-op-wh7nq7n4-206af-q9jvw-worker-a-rdxtm",
+      "locator": "ns/openshift-machine-config-operator node/ci-op-wh7nq7n4-206af-q9jvw-worker-a-rdxtm hmsg/94b2a62d8a",
```

Comment:
I don't know if it matters, but, the 3 other additions to this file also have a locator but they don't contain the `hmsg/.....`.